physx/src/articulation_link.rs

```diff
@@ -235,16 +236,12 @@ impl ArticulationLinkBuilder {
         &self,
         body: &mut ArticulationReducedCoordinate,
         parent: PartHandle,
-        joint_transform: Option<glm::Mat4>,
+        joint_transform: Option<Mat4>,
```

Comment:
Hello, author of glam here being nosy. Beware that due to 16 byte alignment putting a Mat4 in an Option wastes a lot of space.

Reply:
Hey, feedback is always appreciated :). And also a big thanks for writing glam, I am having a very pleasant experience so far.

Yeah 16bit alignment here seems a bit wasteful, I assume it will then become 80bytes.

Reply:
I think we might as well just pass in matrices directly and use `Mat4::identity` instead of `None`? @tgolsson Any thoughts?

Reply:
Probably doesn't matter much if it's not hot code. My experience was `Mat4::inverse` originally returned `Option<Mat4>` because its idiomatic, but micro-benchmarking showed that returning as `Option` took ~80ns versus returning just a `Mat4` taking ~20ns. I tried removing the condition (e.g. always return `Some(Mat4)`) to see if that caused the slow down, but it seemed to just be the `Option`. Micro-benchmarks may not reflect real world performance, but all the same I would recommend keeping an eye on using `Option` with `Mat4` and potentially other aligned glam types in performance sensitive areas.